### PR TITLE
doc/rados/operations: document `ceph balancer status detail`

### PR DESCRIPTION
**Pull request by ljflores**
Followup to https://github.com/ceph/ceph/pull/54801

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -88,6 +88,10 @@ CephFS: Disallow delegating preallocated inode ranges to clients. Config
   cannot repair the dates of existing object locks. Such objects can be identified
   with a HeadObject request based on the x-amz-object-lock-retain-until-date
   response header.
+* RADOS: A detailed version of the `balancer status` CLI command in the balancer
+  module is now available. Users may run `ceph balancer status detail` to see more
+  details about which PGs were updated since the balancer's last optimization.
```
Comment (anthonyeleven) [inline PendingReleaseNotes:93]: since, or in?

**Comment by anthonyeleven** [comment]
bumping because year end

File: PendingReleaseNotes (hunk 1 above)
Comment (ljflores) [reply PendingReleaseNotes:93]: In works.

Comment (ljflores) [reply PendingReleaseNotes]: Fixed!

**Comment by zdover23** [comment]
@ljflores: You gotta get the Signed-off-by line in order.

**Review by anthonyeleven — APPROVED** [review]

**Comment by ljflores** [comment]
> @ljflores: You gotta get the Signed-off-by line in order.

Ah fixed!

**Comment by ljflores** [comment]
Rebased.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/55263 - Reef backport
https://github.com/ceph/ceph/pull/55264 - Quincy backport

**Comment by ljflores** [comment]
> #55263 - Reef backport #55264 - Quincy backport

@zdover23 I have reverted these here:

- https://github.com/ceph/ceph/pull/55358
- https://github.com/ceph/ceph/pull/55359

**Pull request merged**